changelogs/fragments/6011-osx-defaults-errors.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is more a new feature than a bugfix IMO.
```suggestion
minor_changes:
```